docs/docs/guides/providers/credentials-provider.md

```diff
@@ -70,7 +71,8 @@ providers: [
         // If you return null then an error will be displayed advising the user to check their details.
         return null
 
-        // You can also Reject this callback with an Error thus the user will be sent to the error page with the error message as a query parameter
+        // You cannot reject or throw an error from here due to security issues as explained here
```

Comment:
Lets modify both of these to make it a bit more clear.. I'd replace what you have there with something like this maybe?

`
        // It is not recommended to forward specific authentication errors to the user by throwing here as it may leak unnecessary information to the user, for more information please see [here](https://github.com/nextauthjs/next-auth/issues/5839#issuecomment-1321112134)
`